[EV-4429] Update dest_domains desc, and felix DestDomainsByClient. (#1196)
tigera/docs: calico-enterprise/visibility/elastic/flow/datatypes.mdx

| `dest_service_port`               | keyword           | Port name of the destination service.<br />A `-` means :<br />- the original destination did not correspond to a known Kubernetes service (e.g. a services ClusterIP), or<br />- the destination port is aggregated.<br />A `*` means there are multiple service port names matching the destination port number.                        |
| `dest_type`                       | keyword           | Destination endpoint type. Possible values:<br />- `wep`: A workload endpoint, a pod in Kubernetes.<br />- `ns`: A Networkset. If multiple Networksets match, then the one with the longest prefix match is chosen.<br />- `net`: A Network. The IP address did not fall into a known endpoint type.                                     |
| `dest_labels`                     | array of keywords | Labels applied to the destination pod. A hyphen indicates aggregation.                                                                                                                                                                                                                                                                   |

> | `dest_domains`                    | array of keywords | This information pertains to top-level domains linked to the destination IP. It specifically applies to flows reported from the source to destinations outside the cluster. When flowLogsDestDomainsByClient is enabled, the destination domain IPs have been queried by the source IP reported in the flow log. Otherwise, when flowLogsDestDomainsByClient is not enabled, dest_domains: ["A"] doesn't guarantee that the flow corresponds to a connection with domain name A. In either case, it's possible that this destination IP is also linked to other domain names not yet captured by Calico.|

Reviewer: ```suggestion
| `dest_domains`                    | array of keywords | Information on top-level domains linked to the destination IP. Applies specifically to flows reported from the source to destinations outside the cluster. If flowLogsDestDomainsByClient is enabled, the destination domain IPs are queried by the source IP reported in the flow log. If flowLogsDestDomainsByClient is not enabled, dest_domains: ["A"] doesn't guarantee that the flow corresponds to a connection with domain name A. Note that the destination IP may also be linked to other domain names that are not yet captured by Calico (regardless if the parameter is enabled).|
```
